applications/plugins/SofaCarving/examples/SimpleCarving.py

```diff
@@ -0,0 +1,100 @@
+import Sofa
+
+
+#Choose in your script to activate or not the GUI
+USE_GUI = True
+
+
+def main():
+    import SofaRuntime
+    import Sofa.Gui
+
+    root = Sofa.Core.Node("root")
+    createScene(root)
+    Sofa.Simulation.init(root)
+
+    if not USE_GUI:
+        for iteration in range(10):
+            Sofa.Simulation.animate(root, root.dt.value)
+    else:
+        Sofa.Gui.GUIManager.Init("myscene", "qglviewer")
+        Sofa.Gui.GUIManager.createGUI(root, __file__)
+        Sofa.Gui.GUIManager.SetDimension(1080, 1080)
+        Sofa.Gui.GUIManager.MainLoop(root)
+        Sofa.Gui.GUIManager.closeGUI()
+
+def createScene(root):
+    
+    root.gravity=[0, 0, 0]
+    root.dt=0.05
+    root.showBoundingTree = 0
+    root.addObject('RequiredPlugin', name="plug1", pluginName="Sofa.Component.Collision.Detection.Algorithm Sofa.Component.Collision.Detection.Intersection Sofa.Component.Collision.Geometry Sofa.Component.Collision.Response.Contact")
+    root.addObject('RequiredPlugin', name="plug2", pluginName="Sofa.Component.Constraint.Projective Sofa.Component.LinearSolver.Iterative Sofa.Component.ODESolver.Backward")
+    root.addObject('RequiredPlugin', name="plug3", pluginName="Sofa.Component.Engine.Select Sofa.Component.IO.Mesh Sofa.Component.Mass ")
+    root.addObject('RequiredPlugin', name="plug4", pluginName="Sofa.Component.Mapping.Linear Sofa.Component.Mapping.NonLinear Sofa.Component.SolidMechanics.FEM.Elastic Sofa.Component.StateContainer")
+    root.addObject('RequiredPlugin', name="plug5", pluginName="Sofa.Component.Topology.Container.Dynamic Sofa.Component.Topology.Mapping")
+    root.addObject('RequiredPlugin', name="plug6", pluginName="Sofa.Component.Visual Sofa.GL.Component.Rendering3D")
+    root.addObject('RequiredPlugin', name="plug7", pluginName="Sofa.Component.Collision.Detection.Algorithm SofaCarving")
+
+    root.addObject('VisualStyle',displayFlags="")
+    root.addObject('DefaultAnimationLoop')
+    root.addObject('DefaultPipeline',verbose="0")
+    root.addObject('BruteForceBroadPhase')
+    root.addObject('BVHNarrowPhase')
+    root.addObject('DefaultContactManager',response="PenalityContactForceField")
+    root.addObject('MinProximityIntersection',name="Proximity",alarmDistance="0.08",contactDistance="0.05",useSurfaceNormals="false")
+    root.addObject('CarvingManager',active="true",carvingDistance="-0.01")
+  
+ 
+  
+    TT = root.addChild('TT')
+    
+    TT.addObject('EulerImplicitSolver',name="cg_odesolver",printLog="false",rayleighStiffness="0.1",rayleighMass="0.1")
+    TT.addObject('CGLinearSolver',iterations="25",name="linear solver",tolerance="1.0e-9",threshold="1.0e-9")
+    TT.addObject('MeshGmshLoader',filename="mesh/liver.msh",name="loader",scale="1")   
+   
+    TT.addObject('MechanicalObject',template="Vec3d",src="@loader",name="Volume")
+    
+    TT.addObject('TetrahedronSetTopologyContainer', name="topo", src="@loader")
```

Comment:
Instead of using string based path for link connection why not using the linkpath object that was introduced in 
https://github.com/sofa-framework/SofaPython3/pull/265